### ISSUE TITLE
Ignore doesn't work with InputObjectType

### ISSUE THREAD

**Issue by drowhunter** [ChilliCream/hotchocolate#372]
Given the following mutation and InputType
*InputType*
```
public class TransactionInputType : InputObjectType<Transaction>
{
        protected override void Configure(IInputObjectTypeDescriptor<Transaction> descriptor)
        {
            descriptor.Field(f => f.TransactionId).Ignore();
            descriptor.Field(f => f.CreateDate).DefaultValue(DateTime.Now).Ignore();
        }
}
```
*Mutation*
```
public class MutationType : ObjectType<Mutation>
{
        protected override void Configure(IObjectTypeDescriptor<Mutation> descriptor)
        {
            descriptor.Field(f => f.CreateTransaction(default))
               .Type<NonNullType<TransactionGraph>>()
               .Argument("transaction", a => a.Type<NonNullType<TransactionInputType>>());
        }
}
```
```
public class Mutation
{
        private readonly ITransactionRepository _transactionRepository;

        public Mutation(ITransactionRepository transactionRepository)
        {
            _transactionRepository = transactionRepository ?? throw new ArgumentNullException(nameof(transactionRepository));
        }

        public async Task<Transaction> CreateTransaction(Transaction transaction)//, [Service] IEventSender eventSender)
        {
            int id = await _transactionRepository.AddAsync(transaction);
            return transaction;
        }
}
```
The following schema is created

```
{
transactionId: Int!   //should be hidden
createDate: Date!  //should be hidden
name: String = null
amount: Float!
accountId: Int!
billId: Int = null
tagId: Int = null
account: AccountInput
bill: BillInput
tag: TagInput
}
```

**Comment by michaelstaib** [comment]
Thank you for that detailed input ... I will write a test and see what the issue is.

**Comment by michaelstaib** [comment]
OK, I got a test that now proves that Ignore is ignored ;)

**Comment by michaelstaib** [comment]
The bug is now fixed with #373
